rpm/SOURCES/openresty.init

```diff
@@ -73,7 +73,7 @@ configtest() {
 }
 
 rh_status() {
-    status $nginx
+    status -p "$pidfile" $nginx
```

Comment:
I grepped on the internet. 
Unluckily, I could not find any standard docs about init.d/function.
But `status -p "$pidfile"` is used in some famous open source projects' init script.

For example, if `-p` option is not portable, docker is in trouble now...
https://github.com/docker/docker/blob/b248de7e332b6e67b08a8981f68060e6ae629ccf/contrib/init/sysvinit-redhat/docker#L108